docs/README.md

```diff
@@ -277,6 +280,8 @@ environment variable to `YES` before the package build:
 or the UI code is changed. If you wouldn't like to build the UI code you can
 set the `BUILD_UI_DIST` environment variable to `NO` before the package build:
 `BUILD_UI_DIST=NO make package`.
+- Use `make standalone_package` instead of `make package` to avoid
+manually activating the environment before running CodeChecker.
```

Comment:
```suggestion
- Use `make standalone_package` instead of `make package` to avoid
having to manually activate the environment before running CodeChecker.
```

Previous wording would indicate that running CodeChecker would itself activate the environment, and this is something the user does not want to happen, whereas what you are trying to say is that the user could just not have this action-dependency.